webClient/src/app/app.component.ts

```diff
@@ -45,26 +45,37 @@ export class AppComponent {
   }
 
   ngOnInit() {
+    let activeZluxEditors : number = +window.localStorage.getItem("activeZluxEditors")
+    activeZluxEditors > 0 ? activeZluxEditors++ : activeZluxEditors = 1 
+    window.localStorage.setItem("activeZluxEditors",activeZluxEditors.toString())
```

Comment:
this should have a key that starts with the ID of the plugin, to guarantee uniqueness.

Reply:
changed key to `org.zowe.editor-openWindows`